include/gdsync/core.h

```diff
@@ -113,8 +102,7 @@ struct gds_qp {
 
 /* \brief: Poll a peer-enabled CQ.
  *
- * It works similar to ibv_poll_cq but may return EBUSY if a peek entry
- * associated with the CQE is still busy.
+ * It works similar to ibv_poll_cq while taking peer peek entry into account.
```

Comment:
preexisting, similar = similarly

Reply:
"while blocking CPU access to the CQ entries until the GPU had a chance to observe them"

Reply:
Fixed